File: Easy/12.VowelCount/solve.rb
def VowelCount(str)
	vowels = ["a", "e", "o", "i", "u"]
	vowels.reduce(0) { |occurences, vowel| occurences += str.count(vowel) }
end
   
# keep this function call here    
puts VowelCount(STDIN.gets)  

File: Hard/01.PatternChaser/solve.py
def PatternChaser(s):
	occurences = {}
	
	for i in range(0, len(s)-1):
		check = s[i]
		for j in range(i+1, len(s)):
			check += s[j]
			
			if s.count(check) >= 2:
				occurences[check] = s.count(check)
	
	if len(list(occurences.keys())) == 0:
		return "no null"
		
	for key in sorted(occurences, key = lambda l: len(l), reverse=True):
		return "yes " + key
    
# keep this function call here  
print PatternChaser(raw_input())